### PR TITLE
Fixed: run command doesn't exit when using --justlaunch option

### PR DESCRIPTION
**Pull request by tzraikov**

### CODE REVIEW

File: lib/services/livesync/livesync-service.ts [hunk 1]
```diff
@@ -109,6 +109,9 @@ class LiveSyncService implements ILiveSyncService {
 				this.$hooksService.executeBeforeHooks('watch').wait();
 				this.partialSync(liveSyncData[0].syncWorkingDirectory, watchForChangeActions);
 			}
+			if (this.$options.justlaunch) {
+				process.kill(process.pid);
```
Comment (rosen-vladimirov) [inline lib/services/livesync/livesync-service.ts]: This is really strange. What's the idea. Isn't this process.exit? What keeps the process alive?

Comment (tzraikov) [inline lib/services/livesync/livesync-service.ts]: No, this isn't process.exit. Process.exit exits the app unconditionally. This one simulates Ctrl+C and all ongoing tasks registered on SIGKILL will execute.

Comment (tzraikov) [inline lib/services/livesync/livesync-service.ts]: I suggest to polish this code after release.

**Comment by slavchev** [comment]
just for reference https://github.com/NativeScript/nativescript-cli/pull/2396

**Pull request merged**